src/client/driver/command-executors/client-functions/selector-executor/index.js

```diff
@@ -40,6 +41,12 @@ export default class SelectorExecutor extends ClientFunctionExecutor {
         }
     }
 
+    _extendSnapshot (snapshot, node, extensions) {
+        Object.keys(extensions).forEach(prop => {
+            snapshot[prop] = extensions[prop](node);
```

Comment:
It would be good to catch errors from extension methods and report them appropriately, e.g.:
```js
`someProp` custom selector DOM property error:
[error text]
```